.pre-commit-config.yaml

```diff
@@ -3,7 +3,7 @@
 # See https://github.com/pre-commit/pre-commit-hooks/blob/master/.pre-commit-config.yaml for an example with more hooks
 
 ci:
-  autofix_prs: false
+  autofix_prs: true
   autoupdate_commit_msg: "ENH: Autoupdate hooks [pre-commit.ci]"
   autoupdate_schedule: monthly
```

Comment:
do we still need the monthly update?

Reply:
It's like Dependabot: it looks for newer versions of the hooks monthly (it doesn't run `pre-commit` monthly).

Reply:
So, we don't _need_ it, but I think it's a good feature to enable.